### DISCUSSION TITLE
How can the entry/default target be discovered?

### DISCUSSION THREAD

**Discussion by zplan** [nuke-build/nuke#1042] [Q&A]
How can the entry/default target be discovered at runtime?

Code example:
```
public static int Main() => Execute<Build>(x => x.C);

Target C => _ => _
    .DependsOn(A)
    .DependsOn(B)
    .Triggers(D)
    .Executes(() =>
    {
          ...
    });
```

I would like to discover which entry target was called (by default or with the target parameter on CLI). 'C' in this case. When I look at the 'ExecutionPlan' I can not pick the last one, because the last in this case would be 'D'.
Is there a special property?

Thx.

**Comment by matkoch** [comment]
see build status https://nuke.build/docs/fundamentals/builds/#base-properties
